beacon_node/http_api/src/produce_block.rs

```diff
@@ -95,8 +94,8 @@ pub fn build_response_v3<T: BeaconChainTypes>(
     match accept_header {
         Some(api_types::Accept::Ssz) => Response::builder()
             .status(200)
-            .header("Content-Type", "application/ssz")
```

Comment:
This is a lighthouse specific endpoint that was returning `Content-Type application/ssz` instead of `application/octet-stream`. If that was done on purpose, this change can be reverted